### DISCUSSION TITLE
Battery Saver Mode

### DISCUSSION THREAD

**Discussion by propi62** [devbis/z03mmc#97] [Ideas]
Hello, is this a expected battery life?
I once read that the battery should last about a year with the original firmware.

would it be possible to save some energy by increasing the display refresh/datatransmit timings?
for me personally, a refresh every 30 minutes would be enough. (display and transmit)

![image](https://github.com/devbis/z03mmc/assets/39047802/fdc8cb13-1ff7-45e4-a7aa-faaf0dc1042e)

**Comment by propi62** [comment]
battery still working
![image](https://github.com/devbis/z03mmc/assets/39047802/89e0b433-e361-4a5e-b62c-f6216e54f899)

PS: in a recent zigbee2mqtt update, there are some new settings available: -disable smiley, etc.
This functions are not working with the current firmware.
also no new OTA available in zigbee2mqtt.

**Reply by devbis** [reply]
The convertor was modified for a different firmware by a user. I've fixed it back and waiting for PR to merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/7015

**Comment by GoodnessJSON** [comment]
I would appreciate this so much. This firmware absolutely chews batteries currently. Having a custom reporting frequency would be invaluable!

**Comment by propi62** [comment]
i turned the display off. i will report back.
//edit
I also changed the Reporting Settings and hope this will work like this:
![image](https://github.com/devbis/z03mmc/assets/39047802/585ee87a-405e-4b97-812e-9b631d0d90ab)

//edit2 the devices are reporting very often. why is this? with my settings the minimal reporting Intervall should be 600 seconds.

**Comment by cichy63** [comment]
I look forward for any followup - definitely some settings aiming battery saving would be very nice to have

**Comment by gado4525** [comment]
Did you get any positive results with your updated report values? I don't know why developers here doesn't care about battery life on their fw, for me its unusable at this discharge rate.

**Comment by propi62** [comment]
battery life is great with default reporting settings with all zigbee firmwares (i think about 1 year)
